go-controller/pkg/metrics/master.go

```diff
@@ -189,6 +189,43 @@ var metricEgressIPCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "The number of defined egress IP addresses",
 })
 
+var metricEgressIPAssignLatency = prometheus.NewHistogram(prometheus.HistogramOpts{
```

Comment:
Flavio, Can you help me understand why you want to expose a metric to record the configuration latency of egress ip? Trying to understand who would be interested in this latency and why.

Reply:
anserwerd offline; but in a nutshell, this was an idea for helping out with the scaling tests of egressip